### ISSUE TITLE
Attempting to download a full youtube playlist of items, only the first item of the playlist gets downloaded

### ISSUE THREAD

**Issue by johnahnase** [yt-dlp/yt-dlp#7533]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I *intentionally* remove or skip any mandatory\* field

### Checklist

- [X] I'm reporting that yt-dlp is broken on a **supported** site
- [X] I've verified that I'm running yt-dlp version **2023.07.06** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/yt-dlp/yt-dlp/wiki/FAQ#video-url-contains-an-ampersand--and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched [known issues](https://github.com/yt-dlp/yt-dlp/issues/3766) and the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [X] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Region

Europe

### Provide a description that is worded well enough to be understood

yt-dlp -vUxf ba* --prefer-free-formats --audio-format flac https://www.youtube.com/watch?v=jrZfKkT0dFQ&list=OLAK5uy_lA9EuM4TyegOQRbSLyIQ94OptQ7eGsPOo

The above command produces the result of only the first item in the playlist getting downloaded.
I've had this happen some weeks since, and all the while finding no other complaints about it, which leaves me suspicious about my own end.

I'm using Pop!_OS on here.

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [X] If using API, add `'verbose': True` to `YoutubeDL` params instead
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
[debug] Command-line config: ['-vUxf', 'ba*', '--prefer-free-formats', '--audio-format', 'flac', 'https://www.youtube.com/watch?v=jrZfKkT0dFQ']
[debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version stable@2023.06.22 [812cdfa06] (debian*)
[debug] Python 3.10.6 (CPython x86_64 64bit) - Linux-6.2.6-76060206-generic-x86_64-with-glibc2.35 (OpenSSL 3.0.2 15 Mar 2022, glibc 2.35)
[debug] exe versions: ffmpeg 4.4.2 (setts), ffprobe 4.4.2, rtmpdump 2.4
[debug] Optional libraries: Cryptodome-3.11.0, brotli-1.0.9, certifi-2020.06.20, mutagen-1.45.1, pyxattr-0.7.2, secretstorage-3.3.1, sqlite3-2.6.0, websockets-9.1
[debug] Proxy map: {}
[debug] Loaded 1851 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Available version: stable@2023.07.06, Current version: stable@2023.06.22
[debug] Downloading _update_spec from https://github.com/yt-dlp/yt-dlp/releases/latest/download/_update_spec
ERROR: As yt-dlp has been installed via apt, you should use that to update.  If you're on a stable release, also check backports.
[youtube] Extracting URL: https://www.youtube.com/watch?v=jrZfKkT0dFQ
[youtube] jrZfKkT0dFQ: Downloading webpage
[youtube] jrZfKkT0dFQ: Downloading ios player API JSON
[youtube] jrZfKkT0dFQ: Downloading android player API JSON
[youtube] jrZfKkT0dFQ: Downloading m3u8 information
[debug] Sort order given by extractor: quality, res, fps, hdr:12, source, vcodec:vp9.2, channels, acodec, lang, proto
[debug] Formats sorted by: hasvid, ie_pref, quality, res, fps, hdr:12(7), source, vcodec:vp9.2(10), channels, acodec, lang, proto, size, br, asr, vext, aext, hasaud, id
[info] jrZfKkT0dFQ: Downloading 1 format(s): 22
[debug] Invoking http downloader on "https://rr4---sn-uxaxovg-5go6.googlevideo.com/videoplayback?expire=1688769998&ei=bkGoZMT0BIqOyAW2oJa4DA&ip=94.234.119.151&id=o-ACfmLDJZ1QcDgfR4kBfVv4re0_rEOM0bTv-6tKfRKOrj&itag=22&source=youtube&requiressl=yes&mh=qr&mm=31%2C29&mn=sn-uxaxovg-5go6%2Csn-5go7ynld&ms=au%2Crdu&mv=m&mvi=4&pl=16&gcr=se&initcwndbps=1905000&spc=Ul2Sq5C2Bn67xJ4vB0OB6WmcvPgRCLY&vprv=1&svpuc=1&mime=video%2Fmp4&cnr=14&ratebypass=yes&dur=351.085&lmt=1615690522797736&mt=1688748058&fvip=5&fexp=24007246%2C24362687&c=ANDROID&txp=2311222&sparams=expire%2Cei%2Cip%2Cid%2Citag%2Csource%2Crequiressl%2Cgcr%2Cspc%2Cvprv%2Csvpuc%2Cmime%2Ccnr%2Cratebypass%2Cdur%2Clmt&sig=AOq0QJ8wRAIgU_os4paFVWf5fcm8ycjZSAesfbk6y6W2dssoHy2h86MCIGAunQ865uykG9Vpz39Bu3ksUluTFXMdeoER2I39iDMY&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpl%2Cinitcwndbps&lsig=AG3C_xAwRQIgaslF2IIQgqSf8uYw9WVw33Ye58MRPwlKo33HkH9PwZ0CIQCzU4GOss-nv152vXDKlsWL6a0A2_SvO5Hw7aInuYxwuw%3D%3D"
[download] Destination: No Not Now [jrZfKkT0dFQ].mp4
[download] 100% of    6.53MiB in 00:00:06 at 1.06MiB/s
[debug] ffmpeg command line: ffprobe -show_streams 'file:No Not Now [jrZfKkT0dFQ].mp4'
[ExtractAudio] Destination: No Not Now [jrZfKkT0dFQ].flac
[debug] ffmpeg command line: ffmpeg -y -loglevel repeat+info -i 'file:No Not Now [jrZfKkT0dFQ].mp4' -vn -acodec flac -movflags +faststart 'file:No Not Now [jrZfKkT0dFQ].flac'
Deleting original file No Not Now [jrZfKkT0dFQ].mp4 (pass -k to keep)
```

**Comment by bashonly** [comment]
> `[debug] Command-line config: ['-vUxf', 'ba*', '--prefer-free-formats', '--audio-format', 'flac', 'https://www.youtube.com/watch?v=jrZfKkT0dFQ']`

the URL in the debug output is for a single video

what likely happened is you did not quote the URL, so your shell split the command at the `&` in the URL, and the playlist portion of the URL query was not passed to yt-dlp

tl;dr: try putting the URL in quotes

**Comment by johnahnase** [comment]
> > `[debug] Command-line config: ['-vUxf', 'ba*', '--prefer-free-formats', '--audio-format', 'flac', 'https://www.youtube.com/watch?v=jrZfKkT0dFQ']`
> 
> the URL in the debug output is for a single video
> 
> what likely happened is you did not quote the URL, so your shell split the command at the `&` in the URL, and the playlist portion of the URL query was not passed to yt-dlp
> 
> tl;dr: try putting the URL in quotes

Thank you. That is it. I didn't have to do that before. I've downloaded many playlists full of items just like this never putting quotes round. Must be the happenstance I never got a problematic character in the URL before.
